src/projects/digital/api/circuit/src/internal/assembly/components/ConstantHighAssembler.ts

```diff
@@ -13,7 +13,10 @@ export class ConstantHighAssembler extends ComponentAssembler {
 
     public constructor(params: AssemblerParams, sim: DigitalSim) {
         super(params, V(1, 1), {
-            "outputs": () => ({origin: V(0.5, 0), dir: V(1, 0)})
+            "outputs": () => ({
+                origin: V(this.size.x / 2 - this.options.defaultBorderWidth, 0),
```

Comment:
You can at least hard-code the `0.5` probably (instead of `this.size.x / 2` -- but can you also add a comment explaining why this is necessary?